Kentico/ems-mvc-components
# Correct StackOverflow links

### Motivation

CONTRIBUTING.md listed one link to StackOverflow site which was not related to this repository as it addressed Kentico Kontent, not Kentico EMS.

### Checklist


### How to test

Ensure that both links preceded by paragraph "To get help with coding and structuring your projects, use StackOverflow to ask questions with one of the following tags:" work and point at `kentico-mvc` and `kentico` tags.
